b2sdk/utils/__init__.py

```diff
@@ -32,6 +32,12 @@ def b2_url_encode(s):
     :return: URL-encoded string
     :rtype: str
     """
+    # Ensure string is unicode
+    if not isinstance(s, str):
+        try:
+            s = str(s)
+        except TypeError as e:
+            raise TypeError('Cannot convert %r to unicode: %s' % (s, e))
```

Comment:
this change is not related to the other change so it should go to a different PR

Reply:
Thanks for the heads; I'll push this to a different PR.

Reply:
Ok, let me know when you remove it from this PR

Reply:
It has been removed from the PR